deployment/docker/vmanomaly/vmanomaly-integration/vmanomaly_config.yml

```diff
@@ -20,4 +20,4 @@ writer:
 monitoring:
```

Comment:
we need to add `sampling_period` to reader block, otherwise we continue receiving unexpected warnings and can't rely on stable behavior